src/test/java/teammates/test/pageobjects/InstructorFeedbackSessionsPage.java

```diff
@@ -563,4 +571,32 @@ public void reloadSessionsList() {
         executeScript("setIsSessionsAjaxSendingFalse();");
         executeScript("$('#ajaxForSessions').submit();");
     }
+
+    public void verifyResendPublishedEmailButtonDoesNotExist(String courseId, String sessionName) {
+        int sessionRowId = getFeedbackSessionRowId(courseId, sessionName);
+        try {
+            getLinkAtTableRow("session-resend-published-email-for-test", sessionRowId);
+            fail("This element should be hidden.");
+        } catch (NoSuchElementException e) {
+            return;
+        }
```

Comment:
This looks better: 

```java
try {
   getLinkAtTableRow("session-resend-published-email-for-test", sessionRowId);
   signalFailureToDetectException("This element should be hidden");
} catch (NoSuchElementException e) {
    ignoreExpectedException();
}
```

Reply:
In `../test/pageobjects`, we have used `fail()` everywhere to verify if particular web element exists. Why make two extra functions then?

Reply:
These functions already exist and have been used in such cases of exceptions testing. 
However this discussion doesn't matter if you are gonna follow this discussion https://github.com/TEAMMATES/teammates/pull/8827#discussion_r195308565